scvi/models/log_likelihood.py

```diff
@@ -132,6 +212,7 @@ def log_zinb_positive(x, mu, theta, pi, eps=1e-8):
     theta: inverse dispersion parameter (has to be positive support) (shape: minibatch x genes)
     pi: logit of the dropout parameter (real support) (shape: minibatch x genes)
     eps: numerical stability constant
+    return_gene_specific: sum on genes if False, return the log likelihood per gene if True
```

Comment:
you might run into a minor conflict with this once we merge Adam's TotalVI first